src/plugins/stacked-barchart/src/module.test.ts

```diff
@@ -0,0 +1,6 @@
+// Just a stub test
+describe('placeholder test', () => {
+  it('should return true', () => {
+    expect(true).toBeTruthy();
+  });
+});
```

Comment:
Oops :D

Reply:
Indeed, there were no tests; I added a couple of them now. Do we need to test also the rendered HTML?

Reply:
I think we need to stick to project's plugin development best practices.

Reply:
Good point) Thanks, added more tests to check the rendered HTML)